erdblick_app/app/parameters.service.ts

```diff
@@ -310,6 +365,11 @@ export class ParametersService {
         return this.parameters.getValue();
     }
 
+    private isSourceOrMetaData(mapLayerNameOrLayerId: string): boolean {
+        const lower = mapLayerNameOrLayerId.toLowerCase();
+        return lower.includes('sourcedata') || lower.includes('metadata');
```

Comment:
Thanks for introducing this utility! Any reason to keep it private? I think there are other places in erdblick which would need it...

Improvement: Can we use `startsWith` (and maybe `sourcedata-`/`metadata-` with dash)? This check is otherwise too permissive imo, we will get weird side effects if anybody puts these strings into their custom layer names. The better solution would be to solve this on the mapget layer info level, but I guess that ship has sailed for now :/

Reply:
And, do we really need lowercase? Just a small computational expense, but again, unnecessarily permissive. Maybe we can even at least have constants in mapget for the prefixes.